scripts/corelli/calibration/utils.py

```diff
@@ -75,6 +78,57 @@ def bank_numbers(bank_selection: str) -> List[str]:
     return [str(n) for n in sorted(banks)]
 
 
+def preprocess_banks(input_workspace: str, output_workspace: str) -> Workspace2D:
+    r"""
+    Clone the input workspace/run and preprocess the histograms for each tube such
+    that the peak finding algorithm can have a better chance of finding the correct
+    wire position.
+
+    Note: the preprocess step needs to be performed after load_banks, which provided
+    the bank selection.
+    :param input_workspace: input workspace name
+    :param output_workspace: output workspace with pre-processed historgrams
+    """
+    CloneWorkspace(InputWorkspace=str(input_workspace), OutputWorkspace=output_workspace)
+
+    # inline function for 1D singal cleaning
+    # NOTE: an inline function here is by choice as it should only be used for data
+    #       pre-processing only
+    def clean_signals(
+        signal1D: np.ndarray,
+        pixels_per_tube: int = PIXELS_PER_TUBE,
+        peak_interval_estimate: int = 15,
```

Comment:
Can you encode this value of 15 in a named constant ? it could be placed next to the other named constants, at the top of the utils.py module

Reply:
A new module level constant is added as suggested.